app/src/main/java/com/example/mytrainingpal/components/MusclePainWidget.kt

```diff
@@ -20,6 +20,7 @@ import androidx.compose.ui.input.pointer.pointerInput
 import androidx.compose.ui.layout.onGloballyPositioned
 import androidx.compose.ui.unit.IntSize
 import androidx.compose.ui.unit.dp
+import com.example.mytrainingpal.components.svgs.getBodyMusclesWithPain
```

Comment:
I am pretty sure this import just broke the master yesterday. Please remove it and try whether the build still works. If it still works, please delete this import

Reply:
If I remove the import,  getBodyMusclesWithPain(soreMuscles) is not found which prevents the app from building and starting

Reply:
but there should be no svgs folder :/ 

The file `BodyMuscles.kt` should be at `app/src/main/java/com/example/mytrainingpal/components/BodyMuscles.kt`

Reply:
I found the issue. It was in my local BodyMuscles.kt :)

Reply:
It's package read ...components.svg

Reply:
yeah was my mistake. I shared the file like this 😓 ....